doc/releasenotes/15_0_0_release_notes.md

```diff
@@ -0,0 +1,7 @@
+# Release of Vitess v15.0.0
+The entire changelog for this release can be found [here](https:/github.com/vitessio/vitess/blob/main/doc/releasenotes/15_0_0_changelog.md).
```

Comment:
Can you create a stub summary file? It can have just something like
```
## Major Changes

 ### Command-line syntax deprecations

 ### New command line flags and behavior

 ### Online DDL changes
```
And then the generated release notes should have that at the top, and the link to the changelog after that content.

Reply:
For sure